pymc3/distributions/multivariate.py

```diff
@@ -1981,45 +2053,17 @@ class CAR(Continuous):
         "Generalized Hierarchical Multivariate CAR Models for Areal Data"
         Biometrics, Vol. 61, No. 4 (Dec., 2005), pp. 950-961
     """
+    rv_op = car
 
-    def __init__(self, mu, W, alpha, tau, sparse=False, *args, **kwargs):
-        super().__init__(*args, **kwargs)
-
-        D = W.sum(axis=0)
-        d, _ = W.shape
-
-        self.d = d
-        self.median = self.mode = self.mean = self.mu = at.as_tensor_variable(mu)
-        self.sparse = sparse
+    @classmethod
+    def dist(cls, mu, W, alpha, tau, sparse=False, *args, **kwargs):
 
         if not W.ndim == 2 or not np.allclose(W, W.T):
```

Comment:
NumPy cannot be used here, because `W` could be a `TensorVariable`.  You'll need to assert symmetry in a symbolic way&mdash;or simply remove the check for now.

Reply:
@aerubanov, this still needs to be resolved.

Reply:
@brandonwillard , Thanks for the reminder, I made the changes.